<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>501</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="downloadLocation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/Users/Shoummo/Downloads</string>
    </property>
   </widget>
   <widget class="QComboBox" name="videoQuality">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="videoQualityFixed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Shonar Bangla</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video Quality</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
